mmv1/products/compute/Interconnect.yaml

```diff
@@ -0,0 +1,385 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Interconnect'
+kind: 'compute#Interconnect'
+description: |
+  Represents an Interconnect resource. The Interconnect resource is a dedicated connection between
+  Google's network and your on-premises network.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Create a Dedicated Interconnect': 'https://cloud.google.com/network-connectivity/docs/interconnect/concepts/dedicated-overview'
+  api: 'https://cloud.google.com/compute/docs/reference/rest/v1/interconnects'
+base_url: 'projects/{{project}}/global/interconnects'
+self_link: 'projects/{{project}}/global/interconnects/{{name}}'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: 'compute#operation'
+    path: 'name'
+    base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+    wait_ms: 10000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'targetLink'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'status'
+    complete: 'DONE'
+    allowed:
+      - 'PENDING'
+      - 'RUNNING'
+      - 'DONE'
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'compute_interconnect_basic'
+    skip_test: true
```

Comment:
What is the procedure for testing these resources? Typically interconnects are very difficult to test

Reply:
We are using an approach that only works for testing Interconnects on internal GCP projects. Resolved offline.